community/contributing/docs_writing_guidelines.rst

```diff
@@ -0,0 +1,345 @@
+.. _doc_docs_writing_guidelines:
+
+Godot Docs Writing Guidelines
+=============================
+
+The Godot community is rich and international. Users come from all
+around the world. Some of them are young, and many aren't native English
+speakers. That's why we must all write using a clear and a common
+language. For the class reference, the goal is to make it easy to read
+for everyone and precise.
+
+In summary, always try to:
+
+1. Use the direct voice
+2. Use precise action verbs
+3. Avoid verbs that end in -ing
+4. Remove unnecessary adverbs and adjectives.
+5. Ban these 8 words
+6. Use explicit references
+7. Abuse 's to show possession
+
+There are 3 rules on how to describe classes:
+
+1. Give an overview of the node in the brief description
+2. Mention what methods return if it's useful
+3. Use "if true" to describe booleans
+
+.. note::
+
+    A technical writer's job is to pack as much information as possible into
+    the smallest and clearest sentences possible. These guidelines will help
+    you work towards that goal.
+
+7 Rules for a Clear English
+---------------------------
+
+
+Use the direct voice
+~~~~~~~~~~~~~~~~~~~~
+
+Use the direct voice when possible. Take the classes, methods, and
+constants you describe as the subject. It's natural to write using the
+passive voice, but it's harder to read and produces longer sentences.
+
+**Don't** use the passive voice:
+
+.. highlight:: markdown
+
+    void edit_set_pivot ( Vector2 pivot )
+    [...] This method **is implemented** only in some nodes that inherit Node2D.
+
+**Do** use the node's name as a noun:
+
+::
+
+    void edit_set_pivot ( Vector2 pivot )
+    [...] Only some Node2Ds **implement** this method.
+
```

Comment:
I think we need a better example here.  There's still a lot of confusion among contributors just what passive vs. direct is.  Maybe a simple example like:

- Passive:
The man *was bitten* by the dog.

- Active:
The dog *bit* the man.